lib/screens/game/ready_to_start_screen.dart

```diff
@@ -278,12 +266,8 @@ class _ReadyToStartScreenState extends State<ReadyToStartScreen> {
                   ],
                 ),
               ),
-              if (widget.appState.currentOpponent == null ||
```

Comment:
I removed these lines, as it would seem, that neither currenOpponent nor their openGame could be null at this point, as currentGame is used from the start of the function